### PR TITLE
Removing old files. Fixing db error

### PR DESCRIPTION
**Pull request by danitico**
In this PR we are using poetry to handle dependencies as well as deploying the package to production in PyPi. We have also fixed an error with psycopg2 (The original code was sending something that is not a database connection object to psycopg2).

### CODE REVIEW

File: apminsight_site24x7_py/instrumentation/dbapi2.py [hunk 1]
```diff
@@ -89,14 +90,7 @@ def get_host_info(method_info, conn_kwargs):
...
         def conn_wrapper(*args, **kwargs):
-            conn = original(*args, **kwargs)
-            if conn is not None:
-                comp = method_info.get(constants.component_str, '')
-                host_info = ConnectionProxy.get_host_info(method_info, kwargs)
-                new_conn = ConnectionProxy(conn, comp, host_info)
-                return new_conn
-
-            return conn
+            return original(*args, **kwargs)
```
Comment (danitico) [inline apminsight_site24x7_py/instrumentation/dbapi2.py:93]: The host info is already in the original connection object

**Comment by javaguirre** [comment]
Just one thing, I'd keep the module name as it was `apminsight` but the repository name is fine. :-)

**Review by javaguirre — CHANGES_REQUESTED** [review]
Change package name back to `apminsight`

File: apminsight/contrib/django/apps.py [hunk 2]
```diff
@@ -1,17 +1,15 @@
-
-
 from django.apps import AppConfig
 from django.conf import settings
 from apminsight.agentfactory import get_agent
 from .wrapper import instrument_middlewares
 
 
-class ApminsightConfig(AppConfig):
+class apminsightConfig(AppConfig):
```
Comment (javaguirre) [inline apminsight/contrib/django/apps.py]: Apminsight?

**Review by javaguirre — CHANGES_REQUESTED** [review]

**Pull request merged**